reana_client/api/client.py

```diff
@@ -1091,3 +1091,33 @@ def get_workflow_retention_rules(workflow, access_token):
             )
         )
         raise Exception(e.response.json()["message"])
+
+
+def prune_workspace(workflow, parameters, access_token):
```

Comment:
What do you think about accepting as parameters `include_inputs` and `include_outputs` instead of the generic `parameters`?